docs/commands.md

```diff
@@ -25,84 +25,84 @@ The value(s) provided for image and registry are persisted to the `func.yaml` fi
 
 ```console
-function deploy [-n <namespace> -p <path> -i <image> -r <registry>]
+func deploy [-n <namespace> -p <path> -i <image> -r <registry>]
 ```
 
 When run as a `kn` plugin.
 
 ```console
-kn function deploy [-n <namespace> -p <path> -i <image> -r <registry>]
+kn func deploy [-n <namespace> -p <path> -i <image> -r <registry>]
 ```
 
 ## `describe`
 
 Prints the name, route and any event subscriptions for a deployed Function. The user may also specify the name of the function to describe. The namespace defaults to the value in `func.yaml` or the namespace currently active in the user's Kubernetes configuration. The namespace may be specified on the command line, and if so this will overwrite the value in `func.yaml`.
 
-Similar `kn` command: `kn service describe NAME [flags]`. This flag provides a lot of nice information not available in `function describe`, such as revisions, age, annotations and labels. This command should be renamed to make it distinct from `kn` - e.g. `function status`.
+Similar `kn` command: `kn service describe NAME [flags]`. This flag provides a lot of nice information not available in `func describe`, such as revisions, age, annotations and labels. This command should be renamed to make it distinct from `kn` - e.g. `func status`.
 
 ```console
-function describe [-f <format> -n <namespace> -p <path>]
+func describe [-f <format> -n <namespace> -p <path>]
 ```
 
 When run as a `kn` plugin.
 
 ```console
-kn function describe [-f <format> -n <namespace> -p <path>]
+kn func describe [-f <format> -n <namespace> -p <path>]
 ```
 
 ## `list`
 
 Lists all deployed functions. The namespace defaults to the value in `func.yaml` or the namespace currently active in the user's Kubernetes configuration. The namespace defaults to the value in `func.yaml` or the namespace currently active in the user's Kubernetes configuration. The namespace may be specified on the command line, and if so this will overwrite the value in `func.yaml`.
 
-Similar `kn` command: `kn service list [name] [flags]`. This command lists all deployed Knative `Services`. As with other `kn` commands that have similar functionality, there is more information and flexibilty in the `kn` command. However, `kn` will return _all_ `Services`, while `function list` will only display the boson Functions that have been deployed. Consider improving the output of the `function list` command so that it is at least as informative as `kn service list`.
+Similar `kn` command: `kn service list [name] [flags]`. This command lists all deployed Knative `Services`. As with other `kn` commands that have similar functionality, there is more information and flexibilty in the `kn` command. However, `kn` will return _all_ `Services`, while `func list` will only display the boson funcs that have been deployed. Consider improving the output of the `func list` command so that it is at least as informative as `kn service list`.
```

Comment:
```suggestion
Similar `kn` command: `kn service list [name] [flags]`. This command lists all deployed Knative `Services`. As with other `kn` commands that have similar functionality, there is more information and flexibilty in the `kn` command. However, `kn` will return _all_ `Services`, while `func list` will only display the boson functions that have been deployed. Consider improving the output of the `func list` command so that it is at least as informative as `kn service list`.
```